setup.py

```diff
@@ -39,6 +39,7 @@ def get_static_files(path):
 install_requires = [
     'tornado>=4.0,<5',
     'python-daemon<3.0',
+    'toml',
```

Comment:
Can this be excluded? Eventually we want to populate this with dependencies needed but it's important that you can run luigi without toml installed.

Reply:
Fixed